Casks/font/font-m/font-maple-mono-nf.rb

```diff
@@ -1,25 +1,34 @@
 cask "font-maple-mono-nf" do
   # Check on next version bump if the `container` stanza can be removed
-  version "6.4"
-  sha256 "7f2fa17546190d6e6790c562ae1926cacded22459eccf0eb9693719d1325e165"
+  version "7.0-beta36"
+  sha256 "c87aa82dd2eff522db9b003a63acd857572b0f912ad9a25e6bcf28c934abb21b"
 
   url "https://github.com/subframe7536/Maple-font/releases/download/v#{version}/MapleMono-NF.zip"
```

Comment:
```suggestion
  url "https://github.com/subframe7536/Maple-font/releases/download/v#{version}/MapleMono-NF-unhinted.zip"
```

It seems like `MapleMono-NF-unhinted` is the correct Nerd Fonts version:

> Unhinted font is used for high resolution screen (e.g. for MacBook). Using "hinted font" will blur your text or make it looks weird.
> In this case, you can choose MapleMono-OTF / MapleMono-TTF / MapleMono-NF-unhinted / MapleMono-NF-CN-unhinted, etc.